.travis.yml

```diff
@@ -2,8 +2,8 @@ language: node_js
 node_js:
   - "4.2.2"
 before_install:
-  - npm install -g grunt-cli bower
+  - npm install -g bower
```

Comment:
Indeed. 
Most of my Pull Requests were made from the same commit in master at the time, which means that some conflicts will happen eventually. I'll try to update the branches to avoid most conflicts.